integration/util/scenario/actor.js

```diff
@@ -32,11 +32,13 @@ class Actor {
   }
 
   toChainAccount() {
-    return { Eth: this.ethAddress() };
+    const returnValue = {};
+    returnValue[this.chain().nameAsPascalCase()] = this.ethAddress();
```

Comment:
Yeah, I think for now having ethAddress is ok but we will ultimately change a lot of this code when flow comes into play. To be honest, I'm a little bit intimidated refactoring something as pervasive as that could be in js without the help of a compiler

Reply:
`Cmd+Shift+F` and JavaScript are sadly good friends.

Reply:
FWIW, we could use TypeScript, but it's a huge burden for tests since it takes a lot more time to write and tests are pretty one-offy. We could technically upgrade if we think it would help us.